Sources/AppCoinsSDK/Domain/Repositories/Transaction/TransactionRepositoryProtocol.swift

```diff
@@ -9,9 +9,9 @@ import Foundation
 
 internal protocol TransactionRepositoryProtocol {
     
-    func getTransactionBonus(address: String, package_name: String, amount: String, currency: Coin, completion: @escaping (Result<TransactionBonus, TransactionError>) -> Void)
+    func getTransactionBonus(address: String, package_name: String, amount: String, currency: String, completion: @escaping (Result<TransactionBonus, TransactionError>) -> Void)
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done